README.md

```diff
@@ -28,6 +28,12 @@
 
 [![License](https://img.shields.io/badge/License-BSD3-lightgrey.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
+**LATEST RELEASE: You are currently on the main branch which tracks
+under-development progress towards the next release. The current release is
+version [2.39.0](https://github.com/triton-inference-server/vllm_backend/tree/r23.10)
+and corresponds to the 23.10 container release on
+[NVIDIA GPU Cloud (NGC)](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tritonserver).**
+
```

Comment:
```suggestion
**LATEST RELEASE: You are currently on the main branch which tracks
under-development progress towards the next release. The latest release
version corresponds to the [latest release](https://github.com/triton-inference-server/server/releases) of Triton Inference Server and to the latest container published on
[NVIDIA GPU Cloud (NGC)](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tritonserver/tags).**

```

Reply:
something like this

Reply:
We should point to  the specific release branch on vLLM backend which would ensure that the instructions from the README.md works. There has been a lot of confusion in following the steps from the main branch. I will fix the link name just in case.